CIP-066/README.md

```diff
@@ -0,0 +1,426 @@
+---
+CIP: 066
+Title: NFT Identity
```

Comment:
```suggestion
Title: NFT Project Identity
```
Making the title more specific will be helpful, since _NFT Identity_ is too general for people to know what it's really about.  _NFT Project Identity_ would match what you describe in the Abstract & Motivation, or perhaps _NFT Collection Identity_ if that's not too specific.

Reply:
I would even suggest _NFT Verification Standard_ as it is the verifier's identity being used, not the NFT's identity.